projects/meeting-point.md

```diff
@@ -11,7 +11,7 @@ We propose to create an application where your group can pin a location on a map
 
 ### Team
 
-[Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md)
+[Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md), [Halil Akin](people/halil-akin.md)
```

Comment:
Those links are broken. They should start with a slash or have a `../`. Please fix. Thanks!